build.gradle

```diff
@@ -78,7 +78,7 @@ check.dependsOn installDist
 
 dependencies {
     compile files("${System.properties['java.home']}/../lib/tools.jar")
-    compile 'org.broadinstitute:gatk:4.alpha.2-67-gce8ce3a-20161024.213859-1'
+    compile 'org.broadinstitute:gatk:4.alpha.2-84-ge13b22f-SNAPSHOT'
```

Comment:
Is this depending on a non-master commit? We don't want protected depending on a branch of public.

Can you change this to `4.alpha.2-83-gc903e35-20161101.190933-1`, which is the latest master (including a fix needed by @LeeTL1220)